src/dsc/Remoting/Startup.cs

```diff
@@ -35,7 +35,7 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory, IApplicationLifetime lifetime)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)

```

Comment:
@Eneuman IWebHostEnvironment env will still be unused do we want to keep it here or remove it if it not used ?

Reply:
@hsubramanianaks Lets remove it since it unsused